src/core_landice/mode_forward/mpas_li_calving.F

```diff
@@ -1187,10 +1195,10 @@ subroutine eigencalving(domain, err)
          ! First calculate the front retreat rate (Levermann eq. 1)
          calvingVelocity(:) = eigencalvingParameter(:) * max(0.0_RKIND, eMax(:)) * max(0.0_RKIND, eMin(:)) ! m/s
 
-
          ! Convert calvingVelocity to calvingThickness
-         call apply_calving_velocity(meshPool, geometryPool, velocityPool, domain, err)
-
+          call apply_front_ablation_velocity(meshPool, geometryPool, velocityPool, applyToGrounded, &
```

Comment:
If you don't want all the new variables above, you can just pass the values into the subroutine.  If you want to pass the values but make it clear what they are, you can do it like `applyToGrounded=.true.`.  You don't need to remove these new variables, but thought I'd mention it.

Reply:
Noted, but not changed.